modules/tinymce/src/themes/silver/test/ts/browser/editor/tooltip/CollectionTooltipTest.ts

```diff
@@ -0,0 +1,248 @@
+import { FocusTools, Keys, Mouse, UiFinder, Waiter } from '@ephox/agar';
+import { context, describe, it } from '@ephox/bedrock-client';
+import { SugarDocument } from '@ephox/sugar';
+import { TinyHooks, TinyUiActions } from '@ephox/wrap-mcagar';
+
+import Editor from 'tinymce/core/api/Editor';
+import CharmapPlugin from 'tinymce/plugins/charmap/Plugin';
+import EmoticonsPlugin from 'tinymce/plugins/emoticons/Plugin';
+
+import * as TooltipUtils from '../../../module/TooltipUtils';
+
+describe('browser.tinymce.themes.silver.editor.TooltipTest', () => {
+  context('Charmap plugin', () => {
+    const hook = TinyHooks.bddSetup<Editor>({
+      base_url: '/project/tinymce/js/tinymce',
+      plugins: 'charmap',
+      toolbar: 'charmap',
+    }, [ CharmapPlugin ], true);
+
+    it(`TINY-9637: Should trigger tooltip when focus is shifted in collection with keyboard`, async () => {
+      const doc = SugarDocument.getDocument();
+      const editor = hook.editor();
+      editor.focus();
+
+      const buttonSelector = 'button[data-mce-name="charmap"]';
+      TinyUiActions.clickOnToolbar(editor, buttonSelector);
+      await TinyUiActions.pWaitForDialog(editor);
+      await FocusTools.pTryOnSelector('Focus should start on', doc, 'input');
```

Comment:
Could be worth having this as separate function so it can be shared between the `it` blocks